presto-hive/src/test/sql/create-test.sql

```diff
@@ -232,3 +232,12 @@ ALTER TABLE presto_test_partition_schema_change REPLACE COLUMNS (t_data DOUBLE);
 
 INSERT OVERWRITE TABLE presto_test_partition_schema_change_non_canonical PARTITION (t_boolean='0')
 SELECT 'test' FROM presto_test_sequence LIMIT 100;
+
+ANALYZE TABLE presto_test_sequence COMPUTE STATISTICS;
```

Comment:
To have column statistics you have to run
`ANALYZE TABLE .... COMPUTE STATISTICS FOR COLUMNS`.